Extension/CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # C/C++ for Visual Studio Code Change Log
 
+## Pull Request
+* Allow the workspace root to be a non-recursive includePath. [#3121](https://github.com/Microsoft/vscode-cpptools/issues/3121)
```

Comment:
Can you move this under the 0.22.0-insiders section below?

Can you re-word it to say something like "Allow `*` in `includePath` to apply to `browse.path`"? Since non-recursive includPath were already supported, they just didn't auto-copy to browse.path.